### ISSUE TITLE
Error occurs; pinning data to Kaggle 

### ISSUE THREAD

**Issue by issactoast** [rstudio/pins#372]
```r
library(pins)
library(palmerpenguins)

# board_register_kaggle(token = "path/to/kaggle.json")

pin(penguins, 
    description = "The penguins data from R",
    board = "kaggle")

Error in kaggle_create_resource(name, description, token, type, metadata, : Resource creation failed: Failed to resolve the necessary information from the request.
```

I tried to test `pin()` to publish the Kaggle but I got the above error. Could you take a look at the error?

**Comment by hadley** [comment]
Can I ask what you're using this for?

**Comment by issactoast** [comment]
Hi, @hadley,

Thanks for the follow-up. I was trying to publish data on kaggle using `pins` since I tried to upload the predictions results from multiple models like [this dataset](https://www.kaggle.com/hiro5299834/tps-apr-2021-label). After that people use them for [ensembling](https://www.kaggle.com/hiro5299834/tps-apr-2021-pseudo-labeling-voting-ensemble/). I can do this manually but I thought it would be awesome to automate the process at that time.

However, it looks more reasonable to have this function in [kaggler](https://github.com/mkearney/kaggler) package not in pin.

**Comment by hadley** [comment]
With #485:

``` r
library(pins)
library(palmerpenguins)

board <- board_kaggle_dataset()
board %>% pin_write(penguins, desc = "The penguins dataset from R")
#> Using `name = 'penguins'`
#> Guessing `type = 'rds'`

board %>% pin_download("hadleywickham1/penguins") %>% readRDS()
#> # A tibble: 344 × 8
#>    species island    bill_length_mm bill_depth_mm flipper_length_mm body_mass_g
#>    <fct>   <fct>              <dbl>         <dbl>             <int>       <int>
#>  1 Adelie  Torgersen           39.1          18.7               181        3750
#>  2 Adelie  Torgersen           39.5          17.4               186        3800
#>  3 Adelie  Torgersen           40.3          18                 195        3250
#>  4 Adelie  Torgersen           NA            NA                  NA          NA
#>  5 Adelie  Torgersen           36.7          19.3               193        3450
#>  6 Adelie  Torgersen           39.3          20.6               190        3650
#>  7 Adelie  Torgersen           38.9          17.8               181        3625
#>  8 Adelie  Torgersen           39.2          19.6               195        4675
#>  9 Adelie  Torgersen           34.1          18.1               193        3475
#> 10 Adelie  Torgersen           42            20.2               190        4250
#> # … with 334 more rows, and 2 more variables: sex <fct>, year <int>
```

<sup>Created on 2021-08-19 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by hadley** [comment]
Closing this since it's fixed in the PR; see #473 for why this probably won't be included the next pins release.
